tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -278,13 +279,71 @@ describe('Target aggregates', () => {
         expect(await contactsPage.getContactInfoName()).to.equal('Prometheus');
         // assert that the activity card exists and has the right fields.
         expect(await contactsPage.getContactCardTitle()).to.equal('Activity this month');
-        await helperFunctions.validateCardFields(['yesterday Prometheus', '18', '15%']);
+        await helperFunctions.validateCardFields(['yesterday Prometheus', moment().format('YYYY-MM'), '18', '15%']);
 
         await browser.back();
         const secondTargetItem =
           await (await targetAggregatesPage.getTargetItem(expectedTargets[1], CURRENT_PERIOD, districtHospital1.name));
         await helperFunctions.assertTitle(secondTargetItem.title, expectedTargets[1].title);
       });
+
+      it('should display targets of current user on home place', async () => {
+        const targetsConfig = [
+          { id: 'a_target', type: 'count', title: { en: 'what a target!' }, aggregate: true },
+          { id: 'b_target', type: 'percent', title: { en: 'the most target' }, aggregate: true },
+        ];
+        const contactSummaryScript = fs
+          .readFileSync(`${__dirname}/config/contact-summary-target-aggregates.js`, 'utf8');
+
+        const clarissa = contactDocs.find(doc => doc.name === NAMES_DH1[0]);
+        const prometheus = contactDocs.find(doc => doc.name === NAMES_DH1[1]);
+        const targets = {
+          'Clarissa': [
+            { id: 'a_target', value: { total: 50, pass: 40 } },
+            { id: 'b_target', value: { total: 20, pass: 10 } }
+          ],
+          'Prometheus': [
+            { id: 'a_target', value: { total: 20, pass: 18 } },
+            { id: 'b_target', value: { total: 40, pass: 6 } }
+          ],
+          [onlineUser.contact.name]: [
+            { id: 'a_target', value: { total: 1, pass: 1 } },
+            { id: 'b_target', value: { total: 1, pass: 1 } }
+          ],
+        };
+
+        const targetsForContact = (contact) => {
+          return helperFunctions.docTags.map(tag => ({
+            _id: `target~${tag}~${contact._id}~irrelevant`,
+            reporting_period: tag,
```

Comment:
Can you also add targets over three months and validate those are not visible in the activity card fields?